libs/features/mas/src/variants/plans.js

```diff
@@ -2,182 +2,227 @@ import { VariantLayout } from './variant-layout';
+        if (price.options.displayPerUnit) price.dataset.displayPerUnit = 'false';
+        if (price.options.displayTax) price.dataset.displayTax = 'false';
+        if (price.options.displayPlanType) price.dataset.displayPlanType = 'false';
+        legal.setAttribute('data-template', 'legal');
+        headingM.appendChild(legal);
     }
 
-    :host([variant='plans']) .wide-footer #stock-checkbox {
-        margin-top: 0;
-    }
-
-    :host([variant='plans']) #stock-checkbox {
-        margin-top: 8px;
-        gap: 9px;
-        color: rgb(34, 34, 34);
-        line-height: var(--consonant-merch-card-detail-xs-line-height);
-        padding-top: 4px;
-        padding-bottom: 5px;
+    get stockCheckbox() {
+        return this.card.checkboxLabel
+            ? html`<label id="stock-checkbox">
+                <input type="checkbox" @change=${this.card.toggleStockOffer}></input>
+                <span></span>
+                ${this.card.checkboxLabel}
+            </label>`
+            : '';
     }
 
-    :host([variant='plans']) #stock-checkbox > span {
-        border: 2px solid rgb(109, 109, 109);
-        width: 12px;
-        height: 12px;
+    connectedCallbackHook() {
+        const match = matchMobile();
+        if (match?.addEventListener)
+            match.addEventListener('change', this.adaptForMobile);
     }
 
-    :host([variant='plans']) footer {
-        padding: var(--merch-card-plans-padding);
-        padding-top: 1px;
+    disconnectedCallbackHook() {
+        const match = matchMobile();
+        if (match?.removeEventListener)
+            match.removeEventListener('change', this.adaptForMobile);
     }
 
-    :host([variant='plans']) .secure-transaction-label {
-        color: rgb(80, 80, 80);
-        line-height: var(--consonant-merch-card-detail-xs-line-height);
-    }
-      
-    :host([variant='plans']) ::slotted([slot='heading-xs']) {
-        max-width: var(--consonant-merch-card-heading-xs-max-width, 100%);
+    renderLayout() {
+        return html` ${this.badge}
+            <div class="body">
+                <slot name="icons"></slot>
+                <slot name="heading-xs"></slot>
+                <slot name="heading-m"></slot>
+                <slot name="annualPrice"></slot>
+                <slot name="priceLabel"></slot>
+                <slot name="body-xxs"></slot>
+                <slot name="promo-text"></slot>
+                <slot name="body-xs"></slot>
+                <slot name="whats-included"></slot>
+                <slot name="callout-content"></slot>
+                ${this.stockCheckbox}
+                <slot name="badge"></slot>
+                <slot name="quantity-select"></slot>
+            </div>
+            ${this.secureLabelFooter}`;
     }
 
-    :host([variant='plans']) #badge {
-        border-radius: 4px 0 0 4px;
-        font-weight: 400;
-        line-height: 21px;
-        padding: 2px 10px 3px;
-    }
-  `;
+    static variantStyle = css`
```

Comment:
i would ideally have avoided refactoring plans card styling in that PR

Reply:
@npeltier This was unintentional. Since I added some lines before and formatted the file, this happened, not sure how I could have avoided it. Sorry though.